doc/source/ray-observability/ray-metrics.rst

```diff
@@ -261,6 +243,26 @@ To do that, you may want to customize the port that metrics gets exposed to a pr
 
 Now, you can scrape Ray's metrics using Prometheus via ``<ip>:8080``.
 
+Alternate Prometheus host location
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You can choose to run Prometheus on a non-default port or on a different machine. When doing so, you should
+make sure that prometheus can scrape the metrics from your ray nodes following instructions :ref:`here <multi-node-metrics>`.
+
+In addition, both Ray and Grafana needs to know how to access this prometheus instance. This can be configured
+by setting the `RAY_PROMETHEUS_HOST` env var when launching ray. The env var takes in the address to access Prometheus.
+
+
+Alternate Grafana host location
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You can choose to run Grafana on a non-default port or on a different machine. If you choose to do this, the
+:ref:`Dashboard <ray-dashboard>` needs to be configured with a public address to that service so the web page
```

Comment:
Same. Can you link this to the Grafana & Using an existing Grafana instance section? I think we can have 2 links here. E.g.,

Also can you mention the default values? (e.g., it is defaulted to localhost:9000)